src/client-api-proxy.ts

```diff
@@ -1,4 +1,4 @@
-import { ApolloClient, HttpLink, InMemoryCache } from '@apollo/client';
+import { ApolloClient, HttpLink, InMemoryCache } from '@apollo/client/core';
```

Comment:
When used in a backend context, need to import from core otherwise the project requires us to have react installed..

Reply:
Good catch, @bassrock!